components/director/README.md

```diff
@@ -5,24 +5,11 @@ The Director exposes GraphQL API.
 ## Development
 
 After you introduce changes in the GraphQL schema, run the `gqlgen.sh` script.
-To run the Director, use the following command:
 
-```
-go run cmd/main.go
-```
-
-### Prerequisites
```

Comment:
I suggest keeping this documentation, people maybe don't want to use the script for some reason...